airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -188,6 +190,9 @@ def message(self) -> str:
         config=ConfigParameter("core", "max_db_retries"),
         renamed_to=ConfigParameter("database", "max_db_retries"),
     ),
+    ConfigChange(config=ConfigParameter("core", "task_runner")),
+    ConfigChange(config=ConfigParameter("core", "enable_xcom_pickling")),
```

Comment:
Should we keep that one? I think we were going to move away from pickling Xcom?

Reply:
Yeah, i think we did it here: https://github.com/apache/airflow/pull/43905.

Reply:
I'm a bit confused here. This PR was created to warn that `core.task_runner` and `core.enable_xcom_pickling` have been removed. Is there anything I missed or misunderstood?

Reply:
Yeah, i think we should add the warning now that we move away from it.
@potiuk do you want any clarification here? I am good.

Reply:
Ah my bad.